config/webpack.dev.config.js

```diff
@@ -121,6 +121,7 @@ module.exports = Merge.smart(commonConfig, {
       REDDIT_URL: 'https://www.reddit.com',
       APPLE_APP_STORE_URL: 'https://www.apple.com/ios/app-store/',
       GOOGLE_PLAY_URL: 'https://play.google.com/store',
+      DATA_API_BASE_URL: 'http://localhost:18381',
```

Comment:
Should this have Discoveries name in the constant? I think it will make it more clear.

Reply:
Yes it should, bad rebase in here!